pages/index.js

```diff
@@ -15,10 +16,14 @@ export default function Home() {
       ...cardData,
       {
         name,
+        id: nanoid(),
       },
     ]);
   }
-
+  function deleteCard(id) {
+    const newHabit = cardData.filter((card) => card.id !== id);
```

Comment:
newHabit is an unfiiting name here, it implies that a new single Habit is created, but in reality you update the habitsList -> you can name it updatedHabitsList